storage/remote/queue_manager.go

```diff
@@ -702,8 +699,8 @@ outer:
 			continue
 		}
 		t.seriesMtx.Lock()
-		lbls, ok := t.seriesLabels[s.Ref]
-		if !ok {
+		lbls := t.seriesLabels[s.Ref]
+		if lbls.Len() == 0 {
```

Comment:
this is no longer needed to change, I will change it back to checking `ok` in the morning